config/initializers/monkey_patches.rb

```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-# Copyright 2015-2017, the Linux Foundation, IDA, and the
-# CII Best Practices badge contributors
-# SPDX-License-Identifier: MIT
-
-Dir[Rails.root.join('lib', 'ext', '**', '*.rb')].each { |file| require file }
```

Comment:
Fair point. I am actually fairly conservative about monkey patching. Yes, they are very useful, but too many of them can easily lead to it becoming impossible to figure out what's going on when there is a bug.  And you're right, we can always restore it later if it turns out we want it.